proxy/backend/src/main/java/org/apache/shardingsphere/proxy/backend/handler/cdc/CDCBackendHandler.java

```diff
@@ -87,8 +87,9 @@ public CDCResponse createSubscription(final CDCRequest request) {
             return CDCResponseGenerator.failed(request.getRequestId(), CDCResponseErrorCode.SERVER_ERROR, "Not find sharding rule");
         }
         Map<String, List<DataNode>> actualDataNodesMap = new HashMap<>();
-        for (String each : tableNames) {
-            actualDataNodesMap.put(each, getActualDataNodes(shardingRule.get(), each));
+        // TODO need support case-insensitive later
+        for (TableName each : createSubscription.getTableNamesList()) {
+            actualDataNodesMap.put(each.getName(), getActualDataNodes(shardingRule.get(), each.getName()));
         }
```

Comment:
Looks it's not completed. e.g.
1, In ImporterFactory, only openGauss is handled
2, Maybe more code blocks need to be handled, include cdc client